src/backend/api/mirrors_update.py

```diff
@@ -304,11 +307,13 @@ def set_repo_status(
         'TIME',
     )
     try:
-        request = requests.get(
-            url=timestamp_url,
+        async with http_session.get(
+            timestamp_url,
             headers=HEADERS,
-        )
-        request.raise_for_status()
+            timeout=30,
```

Comment:
I suggest to make the number as module global constant and use it everywhere

Reply:
Fixed